requirements.txt

```diff
@@ -1,2 +1,3 @@
 docker-py
 dockerfile-parse
+#pygit2
```

Comment:
So I did some googling and it seems someone has solved this problem: https://github.com/uniphil/venvgit2 - it's not a very clean solution, but for tests in travis, it should be ok to use it.
I think if you add `pip install venvgit2` to `.travis.yml` before running `pip install -r requirements.txt` (with `pygit2` uncommented), it should work. Worth a try :)